### ISSUE TITLE
Some systems failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [Lisp-Stat/data-frame#16]
Building with SBCL 2.2.7.28-02bc916fd / ASDF 3.3.5 for quicklisp dist creation.

Trying to build commit id b916229a73bd64d36443d0d1e70774a4155ddf3e

*data-frame* fails to build with the following error:

```
; caught ERROR:
;   READ error during COMPILE-FILE: Package SB-CLTL2 does not exist. Line: 565, Column: 44, File-Position: 23051 Stream: #<SB-INT:FORM-TRACKING-STREAM for "file /home/quicklisp/quicklisp-controller/dist/build-cache/data-frame/0afa85af3d8f216024d8a7bf148125780a7d7725/data-frame-20220926-git/src/data-frame.lisp" {10136D6A93}>
...
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016C8003}>: COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "data-frame" "data-frame">

```

*data-frame/tests* fails to build because of a failure in  _data-frame_.

[Full log here](http://report.quicklisp.org/2022-09-26/failure-report/data-frame.html)

**Comment by Symbolics** [comment]
Fixed in [58ea914](https://github.com/Lisp-Stat/data-frame/commit/58ea91486e9918d08fbbe840835e7d6c92588e8f)

**Comment by Symbolics** [comment]
Assuming this works now. Please reopen if there is a problem.
